docs/atomic-cluster.1.md

```diff
@@ -0,0 +1,65 @@
+% ATOMIC(1) Atomic Man Pages
+% Matthew Barnes
+% April 2016
+# NAME
+atomic-cluster - Manage cluster hosts
+
+# SYNOPSIS
+**atomic cluster [OPTIONS] SUBCOMMAND**
+
+**atomic cluster** will issue commands to a Commissaire server over HTTP.
+
+Commissaire allows administrators of a Kubernetes, Atomic Enterprise or
+OpenShift installation to perform administrative tasks without the need
+to write custom scripts or manually intervene on systems.
+
+# NOTE
+The `cluster` subcommand is only available when the Commissaire client
+module `commctl` is installed.
+
+# OPTIONS
+**-h** **-help**
+  Print usage statement.
+
+# SUBCOMMANDS
+**get cluster** CLUSTER_NAME
+  Report status of the specified cluster.
```

Comment:
>   Maybe we can flatten out the commands we add so that they are at a higher level like:

> atomic cluster get $NAME
> atomic host delete $NAME
> atomic status # maybe this one is a special case where we say atomic manager status

That's going to trample on the `atomic` namespace too much I think.  `atomic host`, for example, is already a thing (wraps `rpm-ostree`).  I'm on board with your first suggestion though, but picking a good name is hard.

One thing that maybe wasn't stated explicitly enough in https://github.com/projectatomic/atomic/issues/329 was that we already have a standalone command-line tool for this named `commctl` and it shares the same command syntax (and code) with what we're proposing for `atomic cluster`.

IOW, `atomic cluster get cluster` == `commctl get cluster`.

But it's not clear to me (or even @ashcrow I think) whether `commctl` is meant to be kept if/when this stuff lands in `atomic`.